libcxx/docs/Status/Cxx2cPapers.csv

```diff
@@ -46,3 +46,20 @@
 "`P2264R7 <https://wg21.link/P2264R7>`__","LWG","Make ``assert()`` macro user friendly for C and C++","Kona November 2023","","",""
 "`P1673R13 <https://wg21.link/P1673R13>`__","LWG","A free function linear algebra interface based on the BLAS","Kona November 2023","","",""
 "","","","","","",""
+"`P2875R4 <https://wg21.link/P2875R4>`__","LWG","Undeprecate ``polymorphic_allocator::destroy`` for C++26","Tokyo March 2024","","",""
```

Comment:
This one seems to have been already implemented. Should it be Nothing to Do?

Reply:
I'm not sure whether we implemented it. Are you sure we just did not never deprecated it?

Reply:
The implementations is just like the changes required by the paper:
https://isocpp.org/files/papers/P2875R4.pdf
![image](https://github.com/llvm/llvm-project/assets/3337419/735f798d-4f23-4d5e-8211-25820c7d26ed)

https://github.com/llvm/llvm-project/blob/133156c1380ef592ad0fb2dd44473f583dbee1b7/libcxx/include/__memory_resource/polymorphic_allocator.h#L113
https://github.com/llvm/llvm-project/blob/133156c1380ef592ad0fb2dd44473f583dbee1b7/libcxx/include/__memory_resource/polymorphic_allocator.h#L169

I guess @philnik777  should know better?

Reply:
Should we mention that [LWG3036](https://cplusplus.github.io/LWG/issue3036) is reverted by [P2875R4](https://wg21.link/p2875r4) in Cxx23Issues.csv?

Reply:
I'll mark LW3036 as "Reverted By P2875R4" and mark P2875R4 as "Done" since 15.0 (which I think is when we first shipped this).